napari/_app_model/_submenus.py

```diff
@@ -22,4 +22,13 @@
             order=None,
         ),
     ),
+    (
+        MenuId.MENUBAR_FILE,
+        SubmenuItem(
+            submenu=MenuId.SAMPLES,
+            title=trans._('Open Sample'),
+            group=MenuGroup.NAVIGATION,
+            order=100,
```

Comment:
I am guessing the intent of this is to put this submenu as the last item within the group `MenuGroup.NAVIGATION` but seems a little hacky....?

Reply:
My understanding is that this is the intended way of using order. At least I mentally process them as "weight" so this number is less arbitrary